blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,78 @@
+.. _administration_hba:
+
+=========================
+Host Based Authentication
+=========================
+
+.. NOTE::
+
+   Host based Authentication is an Enterprise Edition feature.
+   This feature only concerns connections over ``PostgreSQL Wire Protocol``.
+
+This topic explains how to configure client connections and authentication for CrateDB.
+
+When CrateDB is started, the cluster contains one predefined ``superuser``. This user
+is referred to as ``crate``.
+
+By default, CrateDB’s configuration does not contain any host based authentication
+entries. The authentication is therefore disabled, and all users are allowed to connect.
+
+To allow CrateDB access from specific hosts, you need to configure the
+``auth.host_based`` setting in the ``crate.yml`` file.
+
+Allowing Access to CrateDB
+--------------------------
+
+Client access and authentication over the ``PostgreSQL Wire Protocol`` is administered
+by ``auth.host_based`` configuration in the ``crate.yml`` file.
+
+The general format of the ``auth.host_based`` setting is a list of remote client access
+entries, one per line, that users can specify to restrict access to CrateDB. Each entry
```

Comment:
Each entry may contain no, one, or multiple fields. Allowed fields are `user`, `ip` or `cidr`, and `method`.